VERSION.txt

```diff
@@ -1 +1 @@
-27.0.0
+28.0.0
```

Comment:
Can we add a record to the changelog please for completeness

Reply:
this has been done now